Reduce survival pod spawn rate, remove bearcat radio beacon and holopad

MuckerMayhem opened the pull request:
:cl: Mucker
tweak: Removed Bearcat's holopad and distress beacon. Added a radio beacon circuit board and a RCD.
tweak: Reduced Survival Pod's spawn rate so Exploration isn't constantly stuck doing rescues.
/:cl:

Should hopefully force survivors and Bearcat people to fend for themselves more often, and solve the headache of the Torch having to respond to these on a majority of round.

l3lb0t commented:
Could result in what is essentially a softlock for pod survivors on worlds where they can't go outside in their softsuits.

SupermatterTechnicain commented:
I feel like the overarching issue here isn't the ability to send a distress signal to other vessels, but the fact that most bay offships are either built to be rescued or have a high skill-cap to be survivable. I feel like adding a few more survivability features to the offships may help the situation.

Alex6511 commented:
> I feel like the overarching issue here isn't the ability to send a distress signal to other vessels, but the fact that most bay offships are either built to be rescued or have a high skill-cap to be survivable. I feel like adding a few more survivability features to the offships may help the situation.

That's the point, they're meant to be either very difficult or require interaction with the Torch. The game's focus is the Torch, off-ships exist to interact with the Torch, not stay off doing their own thing.

Alex6511 commented:
Per the above, I feel like this is counterproductive.

MuckerMayhem committed:
tweak bearcat setup, reduce survivalpod spawn rate

removed bearcat holopad and distress beacon. added an RCD and circuit board (near the carp so its hard to get) in their place.
